src/sidebar/test/bootstrap.js

```diff
@@ -1,6 +1,22 @@
 // Expose the sinon assertions.
 sinon.assert.expose(assert, { prefix: null });
 
+assert.rejects = async (asyncFn, error) => {
```

Comment:
It might be useful to create a module that exports a function which can be used to patch `assert`, like we do with `sinon.assert.expose` above. This would live in `src/test-util`.

This way we can package it up for re-use in other applications.